integrations/_infra/Makefile

```diff
@@ -11,8 +11,6 @@ define run_checks
 	ruff check . || exit_code=$$?; \
 	echo "Running black"; \
 	black --check . || exit_code=$$?; \
-	echo "Running yamllint"; \
-	yamllint . || exit_code=$$?; \
```

Comment:
why?

Reply:
yamlint isn't availble at integrations, only in main project

Reply:
Didn't exist before (see https://github.com/port-labs/ocean/commit/bdb4dd7febfe46a18c032545e46222109c51ace5#diff-53118e2f6d6f81647184d5bc94a5275ba819e5d999cee17a0e022b8d54541158 and parents) adding a ticket [PORT-11773](https://getport.atlassian.net/browse/PORT-11773)